SpatialGDK/Source/SpatialGDKEditor/Private/Utils/LaunchConfigEditorLayoutDetails.h

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "CoreMinimal.h"
```

Comment:
Can we be explicit instead of using CoreMinimal?

Reply:
I could, although chasing deps seems a bit futile since the first thing the next include (engine header) does is include CoreMinimal too.